frame/staking/src/default_weights.rs

```diff
@@ -0,0 +1,249 @@
+
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Default weights of pallet-staking.
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0-rc6
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
+
+pub struct WeightInfo;
+impl crate::WeightInfo for WeightInfo {
```

Comment:
```suggestion
impl crate::WeightInfo for () {
```

like here: https://github.com/paritytech/substrate/blob/20498e88f85e8ec3d3149ee1a2ae4966816bd678/frame/democracy/src/default_weight.rs#L25

Reply:
Let's resolve this in the first comment. https://github.com/paritytech/substrate/pull/7007#discussion_r483047721